### PR TITLE
No err if update not needed

### LINKED ISSUE #143: inagoctl update throws error in case update is not needed
We might want to think about handling this case as a success?

### PR DESCRIPTION
**Pull request by Nesurion**
fixes #143

### CODE REVIEW

**Comment by Nesurion** [comment]
output look like this now:
`2016-03-22 13:43:47.896 | INFO     | context.Background: Not updating group 'mygroup': (units already up to date)`

**Comment by Nesurion** [comment]
@giantswarm/cena RFR

File: cli/common.go [hunk 1]
```diff
@@ -95,11 +95,16 @@ func maybeBlockWithFeedback(ctx context.Context, bctx blockWithFeedbackCtx) {
 			os.Exit(1)
 		}
 
+		if controller.IsUnitsAlreadyUpToDate(taskObject.Error) {
+			newLogger.Info(ctx, "Not updating group '%s': (%s)", bctx.Request.Group, taskObject.Error.Error())
```
Comment (zeisss) [inline cli/common.go]: why the parantheses around the error text?

Comment (Nesurion) [inline cli/common.go]: To match the other outputs. https://github.com/giantswarm/inago/blob/no-err-if-update-not-needed/cli/common.go#L105

Comment (zeisss) [inline cli/common.go]: but there is no double colon there.

Comment (Nesurion) [inline cli/common.go]: you are absolutely right! ;)

File: cli/common.go [hunk 2]
```diff
@@ -128,7 +128,7 @@ func maybeBlockWithFeedback(ctx context.Context, bctx blockWithFeedbackCtx) {
 	} else {
 		newLogger.Info(
 			ctx,
-			"Succeeded to %s %d %v for group '%s': %v.",
+			"Succeeded to %s %d %v for group '%s'. %v.",
```
Comment (zeisss) [inline cli/common.go]: leave double colon here please

File: cli/common.go [hunk 3]
```diff
@@ -108,7 +108,7 @@ func maybeBlockWithFeedback(ctx context.Context, bctx blockWithFeedbackCtx) {
 			} else {
 				newLogger.Error(
 					ctx,
-					"Failed to %s %d %v for group '%s': %v. (%s)",
+					"Failed to %s %d %v for group '%s'. %v. (%s)",
```
Comment (zeisss) [inline cli/common.go]: move the colon behind the parantheses please. (or are you writing sentences like this?)

**Comment by JosephSalisbury** [comment]
LGTM

**Pull request merged**